core/templates/components/question-difficulty-selector/question-difficulty-selector.directive.html

```diff
@@ -1,10 +1,12 @@
 <md-radio-group ng-model="skillWithDifficulty._difficulty">
   <md-radio-button ng-repeat="rubric in getSkillIdToRubricsObject()[skillWithDifficulty.getId()]"
+                   ng-if="rubric.getExplanations().length > 0"
                    ng-value="availableDifficultyValues[$index]"
                    class="protractor-test-skill-difficulty-<[rubric.getDifficulty().toLowerCase()]>">
     <strong> <[rubric.getDifficulty()]> </strong>
     <md-list>
-      <md-list-item ng-repeat="explanation in rubric.getExplanations()">
+      <md-list-item class="pl-0" ng-repeat="explanation in rubric.getExplanations()">
```

Comment:
Where is the `pl-0` class defined? Is it a predefined class?

Reply:
That is a bootstrap class for spacing.

Reply:
Ah ok.